js/viz/palette.js

```diff
@@ -141,8 +151,8 @@ export function getPalette(palette, parameters) {
             result = palettes[currentPalette()];
         }
     }
-    result = result || null;
-    return type ? result ? result[type].slice(0) : result : result;
+
+    return type ? result[type].slice(0) : result;
```

Comment:
Is there a chance that results in undefined?

Reply:
The result always is object because `currentPalette` always return a palette.